packages/components/src/Spinner/Spinner.css

```diff
@@ -0,0 +1,78 @@
+.spinner {
+  position: relative;
+  pointer-events: none;
+  opacity: 0.6;
+}
+
+.spinner:after {
```

Comment:
Is there a reason to use a pseudo `:after` here instead of adding the styles onto the `.spinner` class?

Reply:
you're right, it's not needed!